src/Serializer/Serializer.php

```diff
@@ -6,14 +6,15 @@
 
 interface Serializer
 {
-    public function serialize(object $event): string;
+    public const OPTION_PRETTY_PRINT = 'pretty_print';
```

Comment:
i think this should move to our implementation since this has no impact on the API for the implementing side.